### PR TITLE
[perms] Compose org updates

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

This is a pre-PR for implementation of `leaveTeam`. 

This change composes individual relationship updates - allows us to reuse the relationships in bigger (public signature) methods to be used outside of the authorizer.

As it stands, the change is semantically equivalent to the current implementation.

## Related Issue(s)

Fixes #

#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold

### CODE REVIEW

File: components/server/src/authorization/authorizer.ts [hunk 1]
```diff
@@ -52,14 +52,8 @@ export class Authorizer {
...
                 updates: [
-                    v1.RelationshipUpdate.create({
-                        operation: v1.RelationshipUpdate_Operation.TOUCH,
-                        relationship: relationship(objectRef("organization", orgID), "owner", subject("user", userID)),
-                    }),
-                    v1.RelationshipUpdate.create({
-                        operation: v1.RelationshipUpdate_Operation.TOUCH,
-                        relationship: relationship(objectRef("organization", orgID), "member", subject("user", userID)),
-                    }),
+                    ...this.removeOrganizationOwnerRoleUpdates(orgID, userID),
```
Comment (svenefftinge) [inline components/server/src/authorization/authorizer.ts]: should be `addOrganization...`

**Review by svenefftinge — APPROVED** [review]
LGTM, minus comment

File: components/server/src/authorization/authorizer.ts (hunk 1 above)
Comment (easyCZ) [reply components/server/src/authorization/authorizer.ts]: fixed

**Pull request merged**